doc/getting_started/getting_started.rst

```diff
@@ -151,44 +151,47 @@ Building without the Zephyr SDK
 
 The Zephyr SDK is provided for convenience and ease of use. It provides
 cross-compilers for all ports supported by the Zephyr OS and does not require
-any extra flags when building applications or running tests.
-
-In addition to cross-compilers, the Zephyr SDK also provides prebuilt
-host tools. To use the SDK host tools alongside a custom or 3rd party
-cross-compiler, keep the ZEPHYR_SDK_INSTALL_DIR environment variable
-set to the Zephyr SDK installation directory.
-
-To build without the Zephyr SDK's prebuilt host tools, the
-ZEPHYR_SDK_INSTALL_DIR environment variable must be unset, and a 3rd party
-cross-compiler must be installed.
+any extra flags when building applications or running tests. In addition to
+cross-compilers, the Zephyr SDK also provides prebuilt host tools.
+
+It is, however, possible to build the Zephyr OS without the SDK.  If you are
```

Comment:
Not sure if we say Zephyr OS.

Maybe just 

possible to build without the Zephyr SDK.

It's implied that we are trying to build zephyr.